srtcore/api.cpp

```diff
@@ -2823,12 +2839,41 @@ uint16_t srt::CUDTUnited::installMuxer(CUDTSocket* w_s, CMultiplexer& fw_sm)
     return sa.hport();
 }
 
+bool srt::CUDTUnited::inet6SettingsCorrect(const sockaddr_any& muxaddr, const CSrtMuxerConfig& cfgMuxer,
```

Comment:
A better name is needed. A function checking if IPv6 settings are correct is not supposed to return `true` on IPv4 address.